pom.xml

```diff
@@ -1,168 +1,185 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.5</version>
+        <version>2.9</version>
     </parent>
 
-    <artifactId>run-selector</artifactId>
+    <artifactId>run-selector-api</artifactId>
     <packaging>hpi</packaging>
-    <name>Run Selector Plugin</name>
-    <version>1.0-SNAPSHOT</version>
-    <url>http://wiki.jenkins-ci.org/display/JENKINS/Run+Selector+Plugin</url>
+    <name>Run Selector API Plugin</name>
+    <version>0.1.0-SNAPSHOT</version>
+    <url>http://wiki.jenkins-ci.org/display/JENKINS/Run+Selector+API+Plugin</url>
 
     <licenses>
-      <license>
-        <name>The MIT license</name>
-        <url>http://www.opensource.org/licenses/mit-license.php</url>
-        <distribution>repo</distribution>
-      </license>
+        <license>
+            <name>The MIT license</name>
+            <url>http://www.opensource.org/licenses/mit-license.php</url>
+            <distribution>repo</distribution>
+        </license>
     </licenses>
 
+    <developers>
+        <developer>
+            <id>alexsomai</id>
+            <name>Alexandru Somai</name>
+            <email>somai.alexandru@gmail.com</email>
+        </developer>
+        <developer>
+            <id>ikedam</id>
+            <name>Ikeda Yasuyuki</name>
+            <email>devld@ikedam.jp</email>
+        </developer>
+    </developers>
+
     <properties>
-      <jenkins.version>1.580.1</jenkins.version>
-      <java.level>6</java.level>
-      <workflow.version>1.4.2</workflow.version>
+        <jenkins.version>1.580.1</jenkins.version>
+        <java.level>7</java.level>
```

Comment:
Why do you want to switch to java7?
Rather it would be better to switch to Jenkins >= 1.609 if you really need Java7.